assessments/0009-in-toto/in-toto-implementation.md

```diff
@@ -0,0 +1,138 @@
+
+Expose parts of the product specification as separate named documents on the website, as:
+   - [System Overview](https://github.com/in-toto/docs/blob/master/in-toto-spec.md#2-system-overview)  (compare content to https://in-toto.io/in-toto/README and current website About - create versions of increasing depth to address to specific audiences)
+   - [Glossary](https://github.com/in-toto/docs/blob/master/in-toto-spec.md#17-terminology) (convert to alphabetized table) 
+   - [Workflow/Personas](https://github.com/in-toto/docs/blob/master/in-toto-spec.md#2-system-overview) (clearly identify user types and point to relevant doc sections)
+2. Create a high-level technical overview on the project home page suitable for evaluators (see [User Roles](#user-roles)). This might be one of the existing overviews, or can be adapted from them.
+3. Create an index or map to existing docs as a temporary guide, preparatory to moving that information into RTD and integrating it into a doc architecture.
+4. Encourage documentation contributions:
+   - Add Documentation as a contribution area, and clarify the process. Contributor docs should: 
+      - List reviewers/approvers for doc changes and additions.
+      - Encourage the use of Documentation and Good First Issue tags for issues and PRs.
+      - Make doc contributors responsible for updating doc roadmaps/ToCs, to reflect any changes they make to the doc structure (adding or moving documents). 
+   - Specify a documentation "champion" to review and approve doc PRs, decide doc issue priorities -- Justin Cappos?
+   - Develop doc standards (style guide, page templates, locations)
+   - Develop doc architecture
+   - Develop and document a doc versioning/update process that includes updating the Doc home page as locations change and doc is added.
+
+## Actionable items
+
+1. Create **Doc home page** 
+
+  The landing page for the [read-the-docs site](https://in-toto.readthedocs.io/en/latest/), which currently lands on the auto-generated Python reference doc, could be expanded and repurposed as the new overall **Doc home page**.
+
+   1.1 To be immediately useful, the landing page should provide a *top-level roadmap* to existing docs. See [Survey of existing doc](https://github.com/jbogarthyde/CNCF-techdocs/main/assessments/0009-in-toto/survey-of-existing-doc.md)
+
+      This is a necessary step in raising the maturity level of this project. The roadmap should initially describe and provide access to:
+   - Specification
+   - Basic demo
+   - Python reference implementation along with its reference docs (which need to move into a sub-directory) 
+   - Overview of the git repo structure.
+
+   1.2 Move the Description and pointer to the Python Reference implementation to an Implementations section, and move the RTD reference docs for it into this section.
+
+   1.3 Create a doc contributors policy requiring that the Doc home page be updated to reflect any changes to the doc locations and structure. 
```

Comment:
I think this will fix the formatting issues
```suggestion
1. Create **Doc home page** 
The landing page for the [read-the-docs site](https://in-toto.readthedocs.io/en/latest/), which currently lands on the auto-generated Python reference doc, could be expanded and repurposed as the new overall **Doc home page**.

   1.1 To be immediately useful, the landing page should provide a *top-level roadmap* to existing docs. See [Survey of existing doc](https://github.com/jbogarthyde/CNCF-techdocs/main/assessments/0009-in-toto/survey-of-existing-doc.md)

      This is a necessary step in raising the maturity level of this project. The roadmap should initially describe and provide access to:
   - Specification
   - Basic demo
   - Python reference implementation along with its reference docs (which need to move into a sub-directory) 
   - Overview of the git repo structure.

   1.2 Move the Description and pointer to the Python Reference implementation to an Implementations section, and move the RTD reference docs for it into this section.

   1.3 Create a doc contributors policy requiring that the Doc home page be updated to reflect any changes to the doc locations and structure.
```